File: delivery-service/delivery/api/src/main/java/org/example/delivery/api/config/health/HealthOpenApiController.java
package org.example.delivery.api.config.health;

import lombok.RequiredArgsConstructor;
import lombok.extern.slf4j.Slf4j;
import org.example.delivery.api.common.rabbitmq.Producer;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@Slf4j
@RestController
@RequestMapping("/open-api")
@RequiredArgsConstructor
public class HealthOpenApiController {

    private final Producer producer;

    @GetMapping("/health")
    public void health() {
        log.info("health call");
//        producer.producer("delivery.exchange", "delivery.key", "hello");
    }
}

File: delivery/api/src/main/kotlin/org/example/delivery/api/config/health/HealthOpenApiController.kt
package org.example.delivery.api.config.health

import org.slf4j.Logger
import org.slf4j.LoggerFactory
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/open-api")
class HealthOpenApiController {

    private val log: Logger = LoggerFactory.getLogger(this.javaClass)

    @GetMapping("/health")
    fun health() {
        log.info("health call")
    }
}